kuma/users/jinja2/socialaccount/base_signup.html

```diff
@@ -0,0 +1,142 @@
+{#- Set "wiki" and "untrusted" for the benefit of Jinja includes that need them. #}
+{% set wiki = False %}
+{% set untrusted = False %}
+
+<!DOCTYPE html>
+<html lang="{% block lang %}{{ LANG }}{% endblock %}" dir="{{ DIR }}" class="no-js">
+
+<head prefix="og: http://ogp.me/ns#">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <script>(function (d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
+    <title>{% block title %}{{ _('Create your MDN profile') }}{% endblock %}</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="robots" content={%- if request.get_host() in settings.ALLOW_ROBOTS_WEB_DOMAINS -%} "index, follow"
+        {%- else -%} "noindex, nofollow" {%- endif -%} />
+
+    {% include "includes/preload.html" %}
+
+    <link rel="home" href="{{ url('home') }}" />
+    <link rel="license" href="#license" />
+
+    {% block page_css %}
+    {% include "includes/style.html" %}
+    {% endblock %}
+
+    {% include "includes/zilla.html" %}
+
+    {%- if settings.MAINTENANCE_MODE %}
+    {% stylesheet 'maintenance-mode-global' %}
+    {%- endif %}
+
+    {#- If the stylesheet exists, include it. Otherwise, does nothing. #}
+    {% stylesheet 'locale-%s' % LANG %}
+
+    {% block site_analytics %}
+    {% include "includes/google_analytics.html" %}
```

Comment:
You should use `includes/react_google_analytics.html` here, and delete the `kuma/users/jinja2/socialaccount/includes/google_analytics.html` file. That will also clear-up the Travis failures (which I see locally as well) since the `kuma/users/jinja2/socialaccount/includes/google_analytics.html` file has typos which are causing the `make localeextract` command to fail (see below).
```suggestion
    {% include "includes/react_google_analytics.html" %}
```

Reply:
In my comment above, I was assuming that you wanted to use the `kuma/users/jinja2/socialaccount/includes/google_analytics.html` file here instead, since it's added in this PR but not used anywhere else as far as I can tell, but perhaps you really wanted [`includes/google_analytics.html`](https://github.com/mdn/kuma/blob/master/jinja2/includes/google_analytics.html), which is the full-blown MDN Google analytics template that considers `request.user`? That's actually fine too, since we never cache any of the `/<locale>/users/*` endpoints (e.g., `/<locale>/users/account/signup`), and always forward all of the cookies (so those endpoints are free to use the wiki-like templates that consider `request.user` within the template).

If that's the case, then ignore my suggested change above, but it seems you should still remove `kuma/users/jinja2/socialaccount/includes/google_analytics.html` since it doesn't seem to be used anywhere else?